src/components/ListView/components/Item/components/Wrapper/Wrapper.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { sizes } from '../../../../../../styles/constants';
+import Hover from '../Hover';
+
+const { KILO, MEGA, GIGA } = sizes;
+const sizeMap = {
+  [KILO]: 'kilo',
+  [MEGA]: 'mega',
+  [GIGA]: 'giga'
+};
+
+const baseStyles = ({ theme }) => css`
+  label: wrapper__item;
+
+  align-items: center;
+  position: relative;
+  cursor: pointer;
+  border-bottom: ${theme.borderWidth.kilo} solid ${theme.colors.n300};
+
+  &:hover {
+    ${Hover} {
+      display: block;
+    }
+  }
+
+  &:first-child {
+    border-top-left-radius: ${theme.borderRadius.mega};
+    border-top-right-radius: ${theme.borderRadius.mega};
+  }
+
+  &:last-child {
+    border-bottom-left-radius: ${theme.borderRadius.mega};
+    border-bottom-right-radius: ${theme.borderRadius.mega};
+  }
+`;
+
+const paddingStyles = ({ theme, padding }) => css`
+  padding: ${theme.spacings[sizeMap[padding]]};
+`;
+
+const selectedStyles = ({ theme, selected }) =>
+  selected &&
+  css`
+    label: wrapper__item--selected;
+
+    background: ${theme.colors.p100};
+  `;
+
+const Wrapper = styled('div')(baseStyles, paddingStyles, selectedStyles);
```

Comment:
If you want to make each list item keyboard focusable, you can apply `tabIndex={0}` to the wrapper here, e.g. as a default prop. Careful! Do NOT use a positive tab index since this messes up the document's source order (that's a basic accessibility rule).

To make the item fully keyboard accessible, you should add a `onKeyDown` event handler to also call the `onClick` handler for `Enter` and `Spacebar` key presses.

Reply:
I've taken care of this in https://github.com/sumup/circuit-ui/pull/314.

Reply:
cc @felixjung